http-clients/src/main/java/com/palantir/remoting/http/FeignClientFactory.java

```diff
@@ -49,13 +51,15 @@ private FeignClientFactory(
             Decoder decoder,
             ErrorDecoder errorDecoder,
             Function<Optional<SSLSocketFactory>, Client> clientSupplier,
-            BackoffStrategy backoffStrategy) {
+            BackoffStrategy backoffStrategy,
+            Request.Options options) {
```

Comment:
The first .of method is still the same and this constructor is private so we should be ok. The only method we may want is a .of with backoffStrategy and no Request.Options but I'm not sure that's necessary.